Code/Editor/ToolbarManager.cpp

```diff
@@ -590,6 +590,34 @@ AmazonToolbar ToolbarManager::GetObjectToolbar() const
     return t;
 }
 
+QMenu* ToolbarManager::GetPlayButtonMenu() const
+{
+    QMenu* playButtonMenu = new QMenu("Play Game");
+
+    playButtonMenu->addAction(m_actionManager->GetAction(ID_VIEW_SWITCHTOGAME_WINDOWED));
+    playButtonMenu->addAction(m_actionManager->GetAction(ID_VIEW_SWITCHTOGAME_FULLSCREEN));
+
+    return playButtonMenu;
+}
+
+void ToolbarManager::SetupPlayButtonMenu()
+{
+    QAction* playWindowedAction = m_actionManager->GetAction(ID_VIEW_SWITCHTOGAME_WINDOWED);
+    QAction* playFullScreenAction = m_actionManager->GetAction(ID_VIEW_SWITCHTOGAME_FULLSCREEN);
+
+    playWindowedAction->setChecked(false);
+    playFullScreenAction->setChecked(false);
+
+    if (gSettings.gameModeEnterFullScreen)
+    {
+        playFullScreenAction->setChecked(true);
+    }
+    else
+    {
+        playWindowedAction->setChecked(true);
+    }
```

Comment:
niy: This could be replaced with

```
playFullScreenAction->setChecked(gSettings.gameModeEnterFullScreen);
playWindowedAction->setChecked(!gSettings.gameModeEnterFullScreen);
```